### DISCUSSION TITLE
terabox电脑alist登录提示不支持此地区 请问怎么解决

### DISCUSSION THREAD

**Discussion by muown123** [AlistGo/alist#7375] [Q&A]
terabox电脑alist登录提示不支持此地区 请问怎么解决

**Comment by muown123** [comment]
Failed init storage: terabox is not yet available in this area

**Reply by yyyytawa** [reply]
挂梯子

**Comment by anwen-anyi** [comment]
要么挂代理启动，要么在非国内服务器搭建

**Comment by iirre** [comment]
2025.3.16 pc客户端彻底登不上了，地区限制，改梯子也不行

**Reply by lizhcil** [reply]
找到解决办法了吗，现在完全登不了了
